tools/parseAndPopulate/loadJsonFiles.py

```diff
@@ -50,6 +50,13 @@ def __init__(self, path):
         if '1661' in path:
             self.xe1661 = load_json_from_url(
                 'http://www.claise.be/CiscoXE1661.json')
+        if '1662' in path:
```

Comment:
Code like this is fine temporarily, but would like to see a better way of doing this. As is, you will need to update code per OS release. Doesn't seem sensible.

Can we think about a longer term fix?

Reply:
I talked with Benoit about this already.. I want him to have a file there with urls that contains json files so I would have to go through them instead of this. I don t like this solution neither but it is temporary..